driver/src/test/java/org/kaazing/k3po/driver/internal/RobotIT.java

```diff
@@ -468,6 +471,129 @@ public void shouldEcho() throws Exception {
     }
 
     @Test
```

Comment:
If you don't mind please do, these test rely on an protected method in Robot only there for them that I would like to delete at some point.